packages/manager/modules/pci/src/projects/project/vouchers/vouchers.html

```diff
@@ -44,4 +45,5 @@
         </oui-column>
         <oui-column data-title="'cpb_vouchers_available_credit_cell' | translate" data-property="available_credit.text" data-sortable></oui-column>
     </oui-datagrid>
+    <pci-project-black-friday-notice class="mt-4 d-block"></pci-project-black-friday-notice>
```

Comment:
Is the `d-block` helper really needed?

We tend to apply margin in the "same" direction.

So I would suggest you to add this margin directly on the `<oui-datagrid>` (`mb-4`)